tests/dags/test_imports.nopy

```diff
@@ -0,0 +1,26 @@
+# using .nopy extension so that the file won't get reformatted
+# because we want to assert that we can parse crappy formatting.
```

Comment:
Why not use `fmt: off` (and `on`)?

Reply:
because I didn't know about it 😄

Reply:
`fmt: off` works for black I think, but there are still other static checks that are going to be unhappy about it...
For instance `ruff` wants to remove all unused imports